src/Durable/OrchestrationContext.cs

```diff
@@ -30,6 +31,9 @@ public class OrchestrationContext
         [DataMember]
         internal HistoryEvent[] History { get; set; }
 
+        [DataMember]
+        public DateTime CurrentUtcDateTime {get; internal set; }
```

Comment:
Just a thought, no need to do anything about it right now:

We are doing something weird here. We are reusing the same class for two entirely different purposes. First of all, we use it for deserializing the data required to start the orchestrator, which is passed to the worker from the host serialized to Json, and this is why we the relevant properties are marked with `DataMember`. But then, we want to expose _some_ properties to the user script  (such as `Input`, `CurrentUtcDateTime`), while hiding other properties. We may want to separate these into two distinct classes, I just don't know how to easily do this right now. But, if you have a good idea, try that.

Again, we don't _have_ to do this right now, we will not block this PR because of that.